### DISCUSSION TITLE
Infinite Loading Searching for Domain Name

### DISCUSSION THREAD

**Discussion by sayus2884** [vercel/community#548] [Help] [answered]
Hello Vercel Team. For some reason, after doing a domain search, I couldn't click on any of the search results. When I hover over the the items, it only displays a loading cursor. I've tried this on Chrome, Edge, and Safari (Mobile).  Any alternatives I can take?

**Comment by swarnava** [comment, marked as answer]
Hi @sayus2884, We just pushed a fix for this. Let us know if you are still facing problem!
